src/modules/transaction/components/TxSummarizer/index.js

```diff
@@ -41,12 +42,12 @@ const TxSummarizer = ({
   };
 
   return (
-    <Box width="medium" className={`${styles.wrapper} ${classNames} summary`}>
+    <Box width="medium" className={`${styles.wrapper} ${className} summary`}>
       {title && (
         <BoxHeader className={`${styles.header} summary-header`}>
-          <TertiaryButton className="cancel-button" onClick={cancelButton.onClick}>
+          {!hasNoTopCancelButton ? (<TertiaryButton className="cancel-button" onClick={cancelButton.onClick}>
```

Comment:
Should we prevent background click?

Reply:
@reyraa There's already an option of navigating back via the "Go back" button so this just removes the back button at the top of the page